taskchampion/src/workingset.rs

```diff
@@ -69,6 +69,7 @@ impl WorkingSet {
 #[cfg(test)]
 mod test {
     use super::*;
+    use pretty_assertions::{assert_eq};
```

Comment:
I'm surprised `cargo fmt` didn't remove the redundant `{ ... }` brackets?

Reply:
Good point. I'm also surprised that didn't cause a CI failure.  I'll sed that out and land.